modules/EvseV2G/iso_server.cpp

```diff
@@ -0,0 +1,2222 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright (C) 2023 chargebyte GmbH
+// Copyright (C) 2023 Contributors to EVerest
+
+#include <inttypes.h>
+#include <json-c/json.h>
```

Comment:
Adds a dependency to libjson-c, but does not use it. Suggested to remove include